matRad_getRotationMatrix.m

```diff
@@ -1,6 +1,6 @@
-function rotMat = matRad_getRotationMatrix(gantryAngle,couchAngle,system)
+function rotMat = matRad_getRotationMatrix(gantryAngle,collimatorAngle,couchAngle,system)
```

Comment:
This additional argument creates most problems at the moment. E.g. matRad_example1_phantom fails already because in matRad_generateStf matRad_getRotationMatrix is called with only gantry and couch angle. While collimator rotation is more related to gantry than couch, I would still suggest to have it as a third optional argument. This would ease compatability.

Reply:
Okay, so I will put this argument on fourth place to keep the previous work flow.

What would you thing, is it better to edit matRad_getRotationMatrix function or change phantom/patient-mat files and add the missing collimator angle?
Because in the DICOM import workflow I also added a read out of collimator angle for this branch.

Reply:
Ah yes, didn't think it through completely.. I would suggest the following:
- make it the third argument (the system argument was considered a "future feature" is never really used so far, and then we have all important rotations after each other)
- I would suggest to keep downwards compatibility, i.e. we need to have checks a.k.a ìsfield(pln.propStf.collimatorAngle)` in the important functions using coordinate transformations (most importantly the stf generation), that then default the angle to zero for the coming computations. Then it works with patient data that in old format. Our incorporated mat files usually don't have pls/stf saved so it's not an issue for them